tests/resources/Harbor-Pages/ToolKit.robot

```diff
@@ -40,7 +40,7 @@ Filter Object
 
 Select Object
```

Comment:
please add more comments for this method as the name is not clear to understand.